configs/relengworker-prod/config.yml

```diff
@@ -37,7 +37,7 @@ worker_types:
     autoscale:
       algorithm: sla
       args:
-        max_replicas: 20
+        max_replicas: 2
```

Comment:
Downgrading this for now until we solve https://bugzilla.mozilla.org/show_bug.cgi?id=1596439 properly.